candidate.html

```diff
@@ -0,0 +1,21 @@
+<tr>
```

Comment:
what does candidate refer to?

Reply:
It's included here: https://github.com/350org/ak_intl_v3/pull/308/files/628769dec562231255112095496cd1043cfde97b#diff-cd10e45c24834797bffea9a06830aa90bbbc125fbb12d9a088d04cf6a6f645d5R156, I believe for donation forms that involve donating to specific candidates